src/app/_mocked/mock-bar.component.ts

```diff
@@ -7,6 +7,8 @@ import { ColourSeriesData } from '../_models';
   template: ''
 })
 export class MockBarComponent {
+  readonly maxNumberBars = 50;
+
   constructor() {
     console.log('create mock bar');
   }
```

Comment:
I believe you forgot this, console.log.

Reply:
It's only a mock file but yes - I've removed it